### DISCUSSION TITLE
Using glam::Vec3 and bevy::prelude::Vec3 interchangeably

### DISCUSSION THREAD

**Discussion by maximsnoep** [bevyengine/bevy#11980] [Q&A] [answered]
Hi,

I have some function `f(a: glam::Vec3)`. When I use a `bevy::prelude::Vec3`, I get the error **"expected `glam::f32::vec3::Vec3`, found `bevy::prelude::Vec3`"**.

I thought we could interchangeably use `glam::Vec3` and `bevy::prelude::Vec3`? Isn't bevy just re-exporting `glam::Vec3`? Am I wrong? Or is there some trick to get around this?

**Comment by alice-i-cecile** [comment, marked as answer]
There's a good chance you have duplicate glam or bevy dependencies in your project. Use `cargo tree` to investigate :)

**Reply by maximsnoep** [reply]
Thank you! Indeed there was a conflict in glam versions... Thank you very much :)
